scripts/ci.go

```diff
@@ -0,0 +1,71 @@
+// +build none
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+const (
+	GOLANGCI_VERSION = "github.com/golangci/golangci-lint/cmd/golangci-lint@v1.23.1"
+)
+
+func main() {
+
+	log.SetFlags(log.Lshortfile)
+
+	if _, err := os.Stat(filepath.Join("scripts", "ci.go")); os.IsNotExist(err) {
+		log.Fatal("should run build from root dir")
+	}
+	if len(os.Args) < 2 {
+		log.Fatal("cmd required, eg: install")
+	}
+	switch os.Args[1] {
+	case "lint":
+		lint()
+	default:
+		log.Fatal("cmd not found", os.Args[1])
+	}
+}
+
+func lint() {
+
+	verbose := flag.Bool("v", false, "Whether to log verbosely")
+
+	// Make sure golangci-lint is available
+	argsGet := append([]string{"get", GOLANGCI_VERSION})
+	cmd := exec.Command(filepath.Join(runtime.GOROOT(), "bin", "go"), argsGet...)
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		log.Fatalf("could not list packages: %v\n%s", err, string(out))
+	}
+
+	cmd = exec.Command(filepath.Join(GOBIN(), "golangci-lint"))
+	cmd.Args = append(cmd.Args, "run", "--config", ".golangci.yml")
+
+	if *verbose {
```

Comment:
why the pointer?

Reply:
hmm, without the pointer it wont work.

Reply:
with `BoolVar` you can remove it
```suggestion
	if verbose {
```

Reply:
I tried and is still required.

Reply:
Oh.... I see... this kind of breaks the compatibility with go13:
```
scripts/ci.go:53:5: invalid indirect of verbose (type bool)
note: module requires Go 1.14
```

I would like to leave it like it is if you dont mind.